cc-check.go

```diff
@@ -189,7 +196,16 @@ func checkKernelModules(modules map[string]kernelModule) error {
 			if value == expected {
 				ccLog.Infof("Kernel module %q parameter %q has correct value", details.desc, param)
 			} else {
-				return fmt.Errorf("kernel module %q parameter %q has value %q (expected %q)", details.desc, param, value, expected)
+				msg := fmt.Sprintf("kernel module %q parameter %q has value %q (expected %q)", details.desc, param, value, expected)
+
+				// this option is not required when
+				// already running under a hypervisor.
+				if param == "unrestricted_guest" && onVMM {
```

Comment:
I agree. This was just the simplest pragmatic solution I could think of that didn't involve re-inventing the world ;) We might need to do that though if we start growing more exceptions like this.